topology/University of Wisconsin–Madison/CHTC/CHTC.yaml

```
@@ -962,4 +962,28 @@ Resources:
     AllowedVOs:
       - OSG
 
+  CHTC-learn-ap:
+    Active: true
+    ContactLists:
+      Administrative Contact:
+        Primary:
+          ID: OSG1000015
+          Name: Aaron Moate
+      Security Contact:
+        Primary:
+          ID: OSG1000015
+          Name: Aaron Moate
```

```suggestion
        Primary:
          Name: Aaron Moate
          ID: OSG1000015
        Secondary:
          ID: OSG1000141
          Name: Joe Bartkowiak
        Tertiary:
          ID: OSG1000003
          Name: Brian Hua Lin
        Tertiary 1:
          ID: OSG1000670
          Name: Theng Vang
      Security Contact:
        Primary:
          Name: Aaron Moate
          ID: OSG1000015
        Secondary:
          ID: OSG1000141
          Name: Joe Bartkowiak
        Tertiary:
          ID: OSG1000003
          Name: Brian Hua Lin
        Tertiary 1:
          ID: OSG1000670
          Name: Theng Vang
```

The rest of the team isn't off the hook!